docs/sign-up.rst

```diff
@@ -0,0 +1,72 @@
+.. _sign-up:
+
+
+===================================
+Create a CrateDB Cloud user account
+===================================
+
+This page shows you how to:
+
+* Create a user account
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _create-account:
+
+Create an account
+=================
+
+Call up the `CrateDB Cloud sign-up page`_ and complete the form.
+
+.. image:: _assets/img/cloud-sign-up-page.png
+
+Once you've completed the form, select *Sign up*.
+
+You'll receive a confirmation email providing your verification code.
+
+Open your email inbox and open the verification email.
+
+.. NOTE::
+
+  If you didn't receive the verification email, select *Resend it* to request a
+  new verification email.
+
+Copy your verification code from the email.
+
+.. image:: _assets/img/confirmation-email-received.png
+
+Paste your verification code into the corresponding field on the sign-up page
+and select *Confirm Account* to complete the CrateDB Cloud user account
+registration.
+
+.. image:: _assets/img/enter-verification-code.png
+
+Once you've signed up, you're automatically logged into CrateDB Cloud.
+
+Your browser should redirect you to the `CrateDB Cloud Console`_.
+
+.. NOTE::
+
+   CrateDB Cloud ships with the Cloud Console, a dedicated web administration
+   interface for interacting with CrateDB Cloud.
+
+   To learn what the different Cloud Console views look like and what its
+   features can do, check out the `Cloud Console Reference`_.
+
+.. image:: _assets/img/cloud-console-after-sign-up.png
```

Comment:
is this the screen you see when you log in?

it's a bit unwelcoming. would be good to explain what's going on here as a lead in to the next subsection